src/tasks/sep/051-op-betanet-set-respected-game-type/addresses.json

```diff
@@ -0,0 +1,16 @@
+{
+  "420110022": {
```

Comment:
Similar to above comment, addresses in this file don't match what's in https://github.com/ethereum-optimism/devnets/blob/main/betanets/u18-beta/u18-beta-1/chain.yaml